File: 30daysOFcode_C/Day 5.c
// program to print table of number n input from user
// author @aryamansinghk
#include<stdio.h>
int main()
{
    int n,i;
    scanf("%d",&n);
    for(i=1;i<=10;i++)
    {
        printf("%d * %d = %d\n",n,i,n*i);
    }
}

File: 30daysOFcode_TypeScript/day5.ts
function loops(n: number): void {
    if (n < 2 || n > 20) {
        return;
    } else {
        for (var i = 1; i <= 10; i++) {
            console.log(n, "x", i, "=", n * i);
        }
    }
}
